src/test/java/api/CirculationRulesEngineAPITests.java

```diff
@@ -61,15 +63,16 @@ private Policy applyLoanPolicy(ItemType itemType, LoanType loanType,
   }
 
   private Policy applyRequestPolicy(ItemType itemType, String requestType,
-      PatronGroup patronGroup, ShelvingLocation shelvingLocation) {
+      PatronGroup patronGroup, Location location, Institution institution) {
```

Comment:
Can you replace the rules/Location.java and rules/Institution.java parameters by a single domain/Location.java parameter?
domain/Location.java contains locationId, institutionId, campusId and libraryId that are needed for CIRC-413 and CIRC-414.

Reply:
Fixed